src/components/card/index.jsx

```diff
@@ -1,8 +1,10 @@
-export default function Card({ title = '', img }) {
+export default function Card({ img }) {
   return (
-    <div className="shop">
-      <span className="card-title">{title}</span>
-      <img src={img} alt="картинка" className="card-img" />
-    </div>
+
+    <button type="button" className="h-32 w-32 flex rounded-3xl  border-none justify-center flex-col items-center cursor-pointer bg-gradient-to-r from-white to-white hover:from-gray-300 hover:to-gray-300">
+
+      <img src={img} alt="картинка" className="object-contain px-1 w-32 h-20" />
+    </button>
```

Comment:
In general button can't be like a big component. It should be piece of your component